.github/workflows/android-e2e.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       SYSTEM_IMAGES: system-images;android-34;google_apis;x86_64
       AVD_NAME: rn-svg-avd
     concurrency:
-      group: android-example-${{ github.ref }}
+      group: android-e2e-example-${{ github.ref }}
```

Comment:
Should we group all E2E tests together? If something fails, we don't want to ignore it anyway.

Reply:
This is concurrency group which will just cancel in-progress runs when new one is scheduled. Unless I missed the point.

Reply:
Yes, that was an open question if we want it to cancel the remaining e2e tests if something is wrong.

Reply:
Hm, if we place the same concurrency group, our test build never runs.

Reply:
> Yes, that was an open question if we want it to cancel the remaining e2e tests if something is wrong.

They won't get cancelled if something is wrong. Whenever a commit is added to a branch, it will just cancel run for previous commit so resources aren't wasted. It doesn't affect other workflows, neither tests.